src/frontend/src/utils/webAuthnErrorUtils.ts

```diff
@@ -1,16 +1,18 @@
 import { displayError } from "$src/components/displayError";
+import { I18n } from "../i18n";
+import copyJson from "./webAuthnErrorUtils.json";
 
-export const WEBAUTHN_CANCEL_TEMPLATE = {
-  title: "Operation canceled",
-  message:
-    "The interaction with your security device was canceled or timed out. Please try again.",
+export const webAuthnCancelTemplate = () => {
+  const i18n = new I18n();
+  return i18n.i18n(copyJson);
```

Comment:
this is not a template (at least not in the lit sense), plus as much as possible we avoid having the components create their own I18n (instead the screens/pages provide the copy). I think there are other ways to solve this but I'm happy to merge this (if we really need to extract I18n for this specific error)